packages/app/models/cow-order/fetcher.ts

```diff
@@ -0,0 +1,17 @@
+import { COW_API_BASE_URL } from "@/models/cow-order/cow-order";
+
+// how should we get chainId?
+const chainId = 100;
+
+const buildUrl = (address: string) =>
+  `${COW_API_BASE_URL[chainId]}/account/${address}/orders/?limit=500`;
+
+export async function getCowOrders(address: string) {
```

Comment:
ChainId should come from who is calling `getCowOrders`, so we need 1 more parameter here

Reply:
Thanks, I'll do that.